pkg/reconciler/scheduling/placement/placement_reconcile_scheduling_test.go

```diff
@@ -88,7 +88,7 @@ func TestPlacementReconciler(t *testing.T) {
 			apibindings: map[logicalcluster.Name][]*apisv1alpha1.APIBinding{logicalcluster.New("root:org:ws"): {
 				bound(validExport(binding("kubernetes", "root:org:negotiation-workspace"))),
 			}},
-			wantPatch:           `{"metadata":{"annotations":{"scheduling.kcp.dev/placement":null}}}`,
+			wantPatch:           `{"metadata":{"annotations":null}}`,
```

Comment:
🔴  this is a blocker. We cannot deletion all annotation without uid+rv pre-conditions.

Reply:
I just had that thought. Will add